remoteappmanager/static/js/admin/vue-components/accounting/RemoveAccountingDialog.js

```diff
@@ -0,0 +1,40 @@
+define([
+  "components/vue/dist/vue",
+  "jsapi/v1/resources"
+], function(Vue, resources) {
+  "use strict";
+  return {
+    template: `
+      <modal>
+          <div class="modal-header"><h4>Remove Accounting</h4></div>
+          <div class="modal-body">Do you want to remove accounting {{ accToRemove.id }}?</div>
+
+          <div class="modal-footer text-right">
+              <button type="button" class="btn btn-default" @click="close">Cancel</button>
+              <button class="btn btn-primary primary" @click="removeAccounting">Remove</button>
+          </div>
+      </modal>
+      `,
+    props: ['accToRemove'],
+    methods: {
+      close: function () {
```

Comment:
No need for a `close` method, you can put this in the template directly to save some lines of code